lib/views/pages/privacy.html

```diff
@@ -0,0 +1,297 @@
+        richten.</p>
+
+    <p>Sie können diese Datenschutzerklärung jederzeit speichern und ausdrucken.</p>
+
+    <h2>2 Allgemeine Nutzung der Webseite</h2>
+    <h3>2.1 Zugriffsdaten</h3>
+
+    <p>Wir sammeln Informationen über Sie, wenn Sie diese Webseite nutzen. Wir erfassen automatisch Informationen über Ihr Nutzungsverhalten
+        und Ihre Interaktion mit uns und registrieren Daten zu Ihrem Computer oder Mobilgerät. Wir erheben, speichern und nutzen
+        Daten über jeden Zugriff auf unser Onlineangebot (sogenannte Serverlogfiles). Zu den Zugriffsdaten gehören Name und
+        URL der abgerufenen Datei, Datum und Uhrzeit des Abrufs, übertragene Datenmenge, Meldung über erfolgreichen Abruf (HTTP
+        response code), Browsertyp und Browserversion, Betriebssystem, Referrer URL (d.h. die zuvor besuchte Seite), IP-Adresse
+        und der anfragende Provider.</p>
+
+    <p>Wir nutzen diese Protokolldaten ohne Zuordnung zu Ihrer Person oder sonstiger Profilerstellung für statistische Auswertungen
+        zum Zweck des Betriebs, der Sicherheit und der Optimierung unseres Onlineangebotes, aber auch zur anonymen Erfassung
+        der Anzahl der Besucher auf unserer Webseite (traffic) sowie zu Umfang und zur Art der Nutzung unserer Webseite und Dienste,
+        ebenso zu Abrechnungszwecken, um die Anzahl der von Kooperationspartnern erhaltenen Clicks zu messen. Aufgrund dieser
+        Informationen können wir personalisierte und standortbezogene Inhalte zur Verfügung stellen und den Datenverkehr analysieren,
+        Fehler suchen und beheben und unsere Dienste verbessern. Wir behalten uns vor, die Protokolldaten nachträglich zu überprüfen,
+        wenn aufgrund konkreter</p>
+
+    <p>Anhaltspunkte der berechtigte Verdacht einer rechtswidrigen Nutzung besteht. IP-Adressen speichern wir für einen begrenzten
+        Zeitraum in den Logfiles, wenn dies für Sicherheitszwecke erforderlich oder für die Leistungserbringung oder die Abrechnung
+        einer Leistung nötig ist, z. B. wenn Sie eines unserer Angebote nutzen. Nach Abbruch des Vorgangs der Bestellung oder
+        nach Zahlungseingang löschen wir die IP-Adresse, wenn diese für Sicherheitszwecke nicht mehr erforderlich sind. IP-Adressen
+        speichern wir auch dann, wenn wir den konkreten Verdacht einer Straftat im Zusammenhang mit der Nutzung unserer Website
+        haben. Außerdem speichern wir als Teil Ihres Accounts das Datum Ihres letzten Besuchs (z.B. bei Registrierung, Login,
+        Klicken von Links etc.).</p>
+
+    <h3>2.2 E-Mail Kontakt</h3>
+    <p>Wenn Sie mit uns in Kontakt treten (z. B. per Kontaktformular oder E-Mail), speichern wir Ihre Angaben zur Bearbeitung der
+        Anfrage sowie für den Fall, dass Anschlussfragen entstehen. Weitere personenbezogene Daten speichern und nutzen wir
+        nur, wenn Sie dazu einwilligen oder dies ohne besondere Einwilligung gesetzlich zulässig ist.</p>
```

Comment:
Es fehlt noch der komplette Mailchimp Teil oder?

Reply:
Haste da was fertiges?

Reply:
Hm, leider nicht wirklich. Hab halt hier so ne Vorlage, wir müssten aber halt eh nochmal schauen, ob wir bei allen Usern einen Consent haben (theoretisch haben wir das für den Newsletter nämlich nicht bei den Usern, die wir von der Konf-Teilnehmerliste übernommen haben).

> The email newsletter offered here is hosted by myself as well and does not track you but I use the Amazon AWS SES service (in Ireland, the only available location under EU jurisdiction) to send the emails.
> 
> By definition, your email address is stored in order to send out emails to you. To ensure that you’re the owner of the provided email address, we send you a confirmation email before subscribing; Unconfirmed email addresses are kept in the database along with the IP address they were signed up from to prevent abuse and keep the servie safe. The following data is stored to comply with the legal obligation to document this “double opt-in” process: the time and IP address of you filling in the form and the time when you click the confirmation link.
> 
> Once subscribed, you’ll receive newsletter emails; the opening of the message or clicks on the links are not tracked (though sometimes, the links included may be non-public links, which allows to count overall clicks from the newsletter; such links are the same for all subscribers, hence not connected to your identity).
> 
> You can cancel your subscription any time in which case you will not receive further email.

Das halt von der WDRL aber ist halt no-tracking, no Mailchimp.

Reply:
Kannste hier eventuell noch mal was deutsches raussuchen? Denke wir können das noch ergänzen.